.storybook/addons/codeEditorAddon/codeAddon.js

```diff
@@ -68,8 +68,8 @@ export const withCodeEditor = makeDecorator({
   name: `withCodeEditor`,
   parameterName: 'myParameter',
   skipIfNoParametersOrOptions: false,
-  wrapper: (getStory, context, { options }) => {
-    const disableThemeToggle = options ? options.disableThemeToggle : false;
+  wrapper: (getStory, context) => {
+    const disableThemeToggle = context &&  (context.name === 'Custom CSS Properties' || context.title.toLowerCase().includes('templating'));
```

Comment:
Can we still have the options object checked so that we can explicitly disable the toggle as we do for the theme-toggle stories. 
It should be disabled if either of the conditions are true